CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.22.1] - unreleased
```

Comment:
Thanks! You can put some release date directly when merging. At least the advantage of small standalone packages is that tagging and releasing is very easy.